lib/main.js

```diff
@@ -608,17 +587,16 @@ function GetSVG(result) {
   //
   //  Add the speech text and mark the SVG appropriately
   //
-  if (data.speakText) {
-    ID++; var id = "MathJax-SVG-"+ID;
-    svg.removeAttribute("aria-label");
-    svg.setAttribute("aria-labelledby",id+"-Title "+id+"-Desc");
-    for (var i=0, m=svg.childNodes.length; i < m; i++)
-      svg.childNodes[i].setAttribute("aria-hidden",true);
-    var node = MathJax.HTML.Element("desc",{id:id+"-Desc"},[result.speakText]);
-    svg.insertBefore(node,svg.firstChild);
-    node = MathJax.HTML.Element("title",{id:id+"-Title"},["Equation"]);
-    svg.insertBefore(node,svg.firstChild);
-  }
+  for (var i=0, m=svg.childNodes.length; i < m; i++)
+    svg.childNodes[i].setAttribute("aria-hidden",true);
+  // Note: if aria-label exists, getSpeech preserved it in speakText
+  // remove aria-label since labelled-by desc is preferred
+  svg.removeAttribute("aria-label");
```

Comment:
Yes because your recommendation had been to use `desc` instead of an aria-label for SVGs.

The aria-label occurs only via core mathjax: if MathML input has an `alttext`, core mathjax adds that as an aria-label. But in that case we used the `alttext` value for `speakText` above and thus the `desc` will be the same as the label. Moving the change upstream would eliminate this part.